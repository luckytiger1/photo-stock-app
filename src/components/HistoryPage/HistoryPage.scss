.history-page {
  background-color: black;
  font-family: 'SF UI Display Light';

  &__title {
    text-align: center;
    color: white;
    padding-top: 20px;
    padding-bottom: 30px;
    h2 {
      font-size: 32px;
      font-weight: 700;
    }
  }
}
