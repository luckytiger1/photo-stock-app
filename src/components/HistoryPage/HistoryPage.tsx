import React from 'react';
import './HistoryPage.scss';
import HistoryList from '../HistoryList/HistoryList';
import GalleryContainer from '../Gallery/GalleryContainer';

const HistoryPage: React.FC = () => {
  return (
    <>
      <div className="history-page">
        <div className="history-page__title">
          <h2>Your requests</h2>
        </div>
        <HistoryList />
      </div>
      <GalleryContainer />
    </>
  );
};

export default HistoryPage;
