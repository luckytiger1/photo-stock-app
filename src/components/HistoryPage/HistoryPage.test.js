import React from 'react';
import { shallow } from 'enzyme';
import { createSerializer } from 'enzyme-to-json';
import HistoryPage from './HistoryPage';

it('<HistoryPage /> renders correctly', () => {
  expect.addSnapshotSerializer(createSerializer({ mode: 'deep' }));
  expect(shallow(<HistoryPage />)).toMatchSnapshot();
});
