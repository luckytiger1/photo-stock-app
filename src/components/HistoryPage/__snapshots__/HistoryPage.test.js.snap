// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HistoryPage /> renders correctly 1`] = `
<Fragment>
  <div
    className="history-page"
  >
    <div
      className="history-page__title"
    >
      <h2>
        Your requests
      </h2>
    </div>
    <withRouter(Connect(HistoryListContainer)) />
  </div>
  <withRouter(Connect(GalleryContainer)) />
</Fragment>
`;
